tools/packaging/static-build/kernel/build.sh

```diff
@@ -18,6 +18,8 @@ DESTDIR=${DESTDIR:-${PWD}}
 PREFIX=${PREFIX:-/opt/kata}
 container_image="${KERNEL_CONTAINER_BUILDER:-$(get_kernel_image_name)}"
 
+sudo docker image pull ubuntu:22.04
```

Comment:
@GabyCT, sorry, I don't think this is the way to go.

When building the kernel, we do:
* Pull valid build container exists in quay.io
  * In case the pull succeeds, it means the image exists and we use it.
  * In case the pull fails, we build a container image with the tag we looked for using the Dockeffile that's present in this very same folder.
 
When we do a `docker build`, it'll already download the image used as `FROM`, so there's no need to explicitly pull it beforehand.

With everything said, do you have more information to share about the issue you faced?

Reply:
@fidencio the problem is when you do a ./install_kata_kernel.sh or a `make kernel...` the script fails with the following error
```
$ make kernel-tarball-build
make[1]: Entering directory '/home/kata/go/src/github.com/kata-containers/kata-containers'
/home/kata/go/src/github.com/kata-containers/kata-containers/tools/packaging/kata-deploy/local-build//kata-deploy-binaries-in-docker.sh  --build=kernel
sha256:a0cbbc63072f3f48cf557e96f262babd426bf4630dabbe87b21304d9c0217c42
Build kata version 3.2.0-alpha0: kernel
INFO: DESTDIR /home/kata/go/src/github.com/kata-containers/kata-containers/tools/packaging/kata-deploy/local-build/build/kernel/destdir
INFO: build kernel
INFO: Kernel version v5.19.2
Error response from daemon: manifest for quay.io/kata-containers/builders:kernel                                                                                                                                                                                                -b1730e4a67c4c635820079268fd84cc5f37bd378-dirty-x86_64 not found: manifest unkno                                                                                                                                                                                                wn: manifest unknown
#1 [internal] load .dockerignore
#1 transferring context: 2B done
#1 DONE 0.0s

#2 [internal] load build definition from Dockerfile
#2 transferring dockerfile: 617B done
#2 DONE 0.0s

#3 [internal] load metadata for docker.io/library/ubuntu:22.04
#3 ERROR: failed to authorize: DeadlineExceeded: failed to fetch anonymous token            
```

and the error is mainly because it does not make a docker image pull of the ubuntu image that is needed to build the kernel

Reply:
I saw this error mainly on different platforms that are behind a proxy and when I am running as a standalone the scripts